oleksandr.chybiskov/Homework 1/Task2.rb

```diff
@@ -0,0 +1,14 @@
+# Завдання 2 
+# => Заданий Hash (тип ключа у кожного елемента різний)
+# {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+# Знайти кількість ключів, значення яких починається на “yes”
+
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+array_1 = hash.keys
+array_2 = Array.new
```

Comment:
I'm pretty sure usage `array_2` is redundant. 
U can use array methods to implement a chained solution. 
`hash.keys.map { |key| key.to_s }.count { |key| # your implementation of `start_with?` condition }`